docs/src/modules/components/AppFrame.js

```diff
@@ -194,7 +194,13 @@ export default function AppFrame(props) {
             <SvgHamburgerMenu />
           </NavIconButton>
           <Box sx={{ display: { md: 'flex', lg: 'none' } }}>
-            <MuiLogoMenu marginLeft />
+            <LogoWithCopyMenu
+              logo={productIdentifier.logo}
+              logoSvgString={productIdentifier.logoSvg}
+              wordmarkSvgString={productIdentifier.wordmarkSvg}
+              smallerMargin
```

Comment:
Is it intended to add the `smallerMargin` or a bad copy past from the `AppNavDrawer`?

If they all use this props we could remove the option